src/CommunityToolkit.Maui.Core/Views/RoundedView.macios.cs

```diff
@@ -4,14 +4,14 @@
 namespace CommunityToolkit.Maui.Core.Views;
 
 /// <summary>
-/// A rounded <see cref="UIStackView"/>
+/// A rounded <see cref="UIView"/>
 /// </summary>
-public class RoundedStackView : UIStackView
+public class RoundedView : UIView
```

Comment:
Friendly reminder, this is a breaking change.

Reply:
what if we create a new class instead of replacing the existing one? In the next major release, we can remove RoundedStackView

Reply:
I'm cool with that. 

If we go this route and create a new class, should mark `RoundedStackView` as `[Obsolete]` in this PR.

Reply:
Thank you @pictos for your review.
@brminnick, put RoundedStackView back and gave it Obsolete.
Certainly this was a breaking change. Should have given Obsolete. 
I would like to be careful in the future.

Thank you everyone.